Sources/Extensions/CoreGraphics/CGVectorExtensions.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  CGVectorExtensions.swift
+//  SwifterSwift
+//
+//  Created by Robbie Moyer on 7/25/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+#if canImport(CoreGraphics)
+import CoreGraphics
+
+// MARK: - Properties
+public extension CGVector {
+
+    /// SwifterSwift: The angle of rotation (in radians) of the vector.
+    /// The range of the angle is -π to π; an angle of 0 points to the right.
+    public var angle: CGFloat {
+        // https://en.wikipedia.org/wiki/Atan2
+        return atan2(dy, dx)
+    }
+
+    /// SwifterSwift: The magnitude (or length) of the vector.
+    public var magnitude: CGFloat {
+        // https://en.wikipedia.org/wiki/Euclidean_vector#Length
+        return sqrt((dx * dx) + (dy * dy))
+    }
+}
+
+// MARK: - Initializers
+public extension CGVector {
+
+    /// SwifterSwift: Creates a vector with the given magnitude and angle.
+    ///
+    ///     let vector = CGVector(angle: .pi, magnitude: 1)
+    ///
+    /// - Parameters:
+    ///     - angle: The angle of rotation (in radians) counterclockwise from the positive x-axis.
+    ///     - magnitude: The lenth of the vector.
+    ///
+    public init(angle: CGFloat, magnitude: CGFloat) {
+        // https://www.grc.nasa.gov/WWW/K-12/airplane/vectpart.html
+        self.init(dx: magnitude * cos(angle), dy: magnitude * sin(angle))
+    }
+}
+
+// MARK: - Operators
+public extension CGVector {
+
+    /// SwifterSwift: Multiplies a scalar and a vector (commutative).
+    ///
+    ///     let vector = CGVector(dx: 1, dy: 1)
+    ///     let largerVector = vector * 2
+    ///
+    /// - Parameters:
+    ///   - vector: The vector to be multiplied
+    ///   - scalar: The scale by which the vector will be multiplied
+    /// - Returns: The vector with its magnitude scaled
+    public static func * (vector: CGVector, scalar: CGFloat) -> CGVector {
+        return CGVector(dx: vector.dx * scalar, dy: vector.dy * scalar)
+    }
+
+    /// SwifterSwift: Multiplies a scalar and a vector (commutative).
+    ///
+    ///     let vector = CGVector(dx: 1, dy: 1)
+    ///     let largerVector = 2 * vector
+    ///
+    /// - Parameters:
+    ///   - scalar: The scalar by which the vector will be multiplied
+    ///   - vector: The vector to be multiplied
+    /// - Returns: The vector with its magnitude scaled
+    public static func * (scalar: CGFloat, vector: CGVector) -> CGVector {
+        return CGVector(dx: scalar * vector.dx, dy: scalar * vector.dy)
+    }
+
+    /// SwifterSwift: Compound assignment operator for vector-scalr multiplication
+    ///
+    ///     var vector = CGVector(dx: 1, dy: 1)
+    ///     vector *= 2
+    ///
+    /// - Parameters:
+    ///   - vector: The vector to be multiplied
+    ///   - scalar: The scale by which the vector will be multiplied
+    public static func *= (vector: inout CGVector, scalar: CGFloat) {
+        // swiftlint:disable shorthand_operator
```

Comment:
You can just use `// swiftlint:disable next shorthand_operator` and then you won't need to re-enable it as it will do it just for the next line. You can do this in all the other places you used this too.